src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -132,25 +144,27 @@ function ExportAnnotations({
           <label htmlFor="export-user" className="block font-medium">
```

Comment:
I suggest changing the ID here and in `ImportAnnotations` to use `useId` to generate the ID. That makes the connection between the label and select more obvious, and also generally avoids potential hazards between conflicting IDs.